CHANGELOG.md

```diff
@@ -44,9 +44,13 @@ Ref: https://keepachangelog.com/en/1.0.0/
 ### Dependencies
 
 - Bump `github.com/cometbft/cometbft` from v0.38.10-pio-1 to v0.38.10 ([#2097](https://github.com/provenance-io/provenance/pull/2097))
+- Bump `cosmossdk.io/client/v2` from 2.0.0-beta.2 to 2.0.0-beta.4 ([#2100](https://github.com/provenance-io/provenance/pull/2100))
 
 ---
 
+### Dependencies
+- Bump `cosmossdk.io/client/v2` from 2.0.0-beta.2 to 2.0.0-beta.4 ([#2100](https://github.com/provenance-io/provenance/pull/2100))
+
```

Comment:
Let's hold off on this until after `v1.19.0`.

Also, this needs to be fixed before merging.

Reply:
All good now.